webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js

```diff
@@ -30,50 +28,48 @@ const Disk = ({
 }) => (
   <div className="disk-container">
     <div className="form-group">
-      <label className="col-md-2 control-label">
-        {__('Disk name')}
-      </label>
-      <div className="col-md-4">
-        {name}
-      </div>
+      <label className="col-md-2 control-label">{__('Disk name')}</label>
+      <div className="col-md-4">{name}</div>
       <div className="col-md-2">
-        {!vmExists &&
-        <Button className="close" onClick={removeDisk}>
-          <span aria-hidden="true">×</span>
-        </Button>}
+        {!vmExists && (
+          <Button className="close" onClick={removeDisk}>
+            <span aria-hidden="true">×</span>
```

Comment:
nitpick: shoudl this be ```&times```

Reply:
Sorry @ohadlevy I'm not sure I understand, can you explain please?